### ISSUE TITLE
Full URIResolver URI not shown in SPARQL Endpoint

### ISSUE THREAD

**Issue by samuell** [rdfio/RDFIO#23]
### How to reproduce:
- Use the rdfio-vagrant box at commit [9f2fc6](https://github.com/rdfio/rdfio-vagrantbox/tree/9f2fc6a4937a3343555ebb0c15dd0d4a37f3be67).
- Install the dependencies
- Run `vagrant up`
- Remember to install the ARC2 tables by logging in (details in vagrant box readme), and going to RDFIO Admin special page and hitting the "Setup" button. 
- Navigate to the SPARQL endpoint.
### What you see:

The text field contains this code:

``` sparql
@PREFIX w : <Special:URIResolver/> .

SELECT *
WHERE { ?s ?p ?o }
LIMIT 25
```

![selection_219](https://cloud.githubusercontent.com/assets/125003/18870251/1a5cd8c0-84b0-11e6-8931-eff414e1cea8.png)

As you can see, a full URI is not shown for the URIResolver, but only the wiki page name (`<Special:URIResolver/>`).
### What is expected

Something like this is expected (notice the full URL, on the first line):

``` sparql
@PREFIX w : <http://localhost:8080/w/index.php/Special:URIResolver/> .

SELECT *
WHERE { ?s ?p ?o }
LIMIT 25
```

**Comment by samuell** [comment]
Seems to be fixed in bada26ed9ec9daa62288fbb52cde8a3137d96e66

Should run some more tests though.

**Comment by samuell** [comment]
Merged, and will be included in v2.0.9. Closing for now.
